src/Sylius/Component/Attribute/AttributeType/SelectAttributeType.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Attribute\AttributeType;
+
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * @author Laurent Paganin-Gioanni <l.paganin@algo-factory.com>
+ */
+class SelectAttributeType implements AttributeTypeInterface
+{
+    const TYPE = 'select';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStorageType()
+    {
+        return AttributeValueInterface::STORAGE_JSON;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getType()
+    {
+        return static::TYPE;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validate(AttributeValueInterface $attributeValue, ExecutionContextInterface $context, array $configuration)
```

Comment:
Why do you have empty function? You have missed something here or violated one of SOLID principles here